src/widgets/Dialogs.js

```diff
@@ -180,7 +180,7 @@ define(function (require, exports, module) {
             }
         } else if (brackets.platform === "mac") {
             // CMD+D Don't Save
```

Comment:
You should rather call this CMD+Delete, right?
Also, I think CMD+D still should trigger Don't Save, shouldn't it?